pkg/cloud/services/compute/instance_test.go

```diff
@@ -28,6 +28,7 @@ func Test_getPortName(t *testing.T) {
 		opts         *infrav1.PortOpts
 		netIndex     int
 	}
+	True := true
```

Comment:
I'm sure this is idiomatic, but 🤦‍♂️

Reply:
This is unfortunately the only way I could create a pointer to a bool without upsetting the GO gods